apis/acmpca/v1alpha1/certificateauthoritypermission_types.go

```diff
@@ -57,10 +57,10 @@ type CertificateAuthorityPermissionParameters struct {
 	// +immutable
 	Actions []string `json:"actions,omitempty"`
 
-	// The AWS Service or identity
-	// +optional
+	// The AWS service or identity that receives the permission. At this
+	// time, the only valid principal is acm.amazonaws.com.
 	// +immutable
-	Principal *string `json:"principal,omitempty"`
+	Principal string `json:"principal"`
```

Comment:
Should we bump this to `v1alpha2`? Or was this previously implicitly required by AWS rejecting if not supplied?

Reply:
It seems like we were using a constant for the value of this field and the field was no-op. So, making it required now would break people who didn't supply a value for this field. I made it optional again but added this as default value. I think we can fix the schema when we bump to v1beta1.